cdap-api/src/main/java/co/cask/cdap/api/workflow/AbstractWorkflow.java

```diff
@@ -126,4 +153,15 @@ protected final void addSpark(String spark) {
     Predicate<WorkflowContext> predicate) {
     return configurer.condition(predicate);
   }
+
+  /**
+   * Adds a condition to the {@link Workflow}.
+   * @param uniqueName the unique name to be assigned to the {@link WorkflowNode} that represents the condition
+   * @param predicate the {@link Predicate} to be evaluated to determine which branch to take
+   * @return the {@link WorkflowConditionConfigurer} to configure the branches in the condition
+   */
+  protected final WorkflowConditionConfigurer<? extends WorkflowConfigurer> condition(
+    String uniqueName, Predicate<WorkflowContext> predicate) {
```

Comment:
now the indentation is really ugly. Why not put the method name and its parameters in one line? Does this even pass checkstyle?